### ISSUE TITLE
'Import all unresolved names' duplicates imports

### ISSUE THREAD

**Issue by BladeMF** [phpactor/phpactor#1835]
When using "import all unresolved names" on this class:
```php
<?php

declare(strict_types=1);

namespace MarsIPM\Tests\Security\Unit\Voter;

use MarsIPM\Tests\Utils\ContainerHelperTrait;
use MarsIPM\Tests\Utils\DatabaseUtils;
use MarsIPM\Tests\Utils\ORMHelperTrait;
use MarsIPM\Tests\Utils\UserHelperTrait;
use Prophecy\PhpUnit\ProphecyTrait;
use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;

class IssueAnalysisEditAdminVoterTest extends KernelTestCase
{
	use ProphecyTrait;
	use ORMHelperTrait;
	use UserHelperTrait;
	use ContainerHelperTrait;

	public function setUp(): void
	{
		self::bootKernel([]);
		DatabaseUtils::resetDatabase();
	}

	public function voter(): IssueAnalysisEditVoter
	{
		return new IssueAnalysisEditVoter();
	}

	public function test_supports_edit_attribute(): void
	{
		self::assertTrue($this->voter()->supportsAttribute(IssueAnalysisEditVoter::ATTRIBUTE));
		self::assertEquals(IssueAnalysisEditVoter::ACCESS_ABSTAIN, $this->voter()->vote(
			$this->prophesize(TokenInterface::class)->reveal(),
			$this->getAnalysis(region: null),
			['some-other-attribute', ]
		));
	}

	public function test_supports_issue_analysis_subject(): void
	{
		self::assertTrue($this->voter()->supportsType(IssueAnalysis::class));
		self::assertEquals(IssueAnalysisEditVoter::ACCESS_ABSTAIN, $this->voter()->vote(
			$this->prophesize(TokenInterface::class)->reveal(),
			new stdClass(),
			[IssueAnalysisEditVoter::ATTRIBUTE, ]
		));
	}

	public function test_no_user_in_token_will_deny_access(): void
	{
		self::assertEquals(
			IssueAnalysisEditVoter::ACCESS_DENIED,
			$this->voter()->vote(
				$this->tokenForUser(null),
				$this->getAnalysis(region: null),
				[IssueAnalysisEditVoter::ATTRIBUTE, ]
			)
		);
	}
}
```
I get these usings:
```php
use MarsIPM\Entity\IssueAnalysis;
use MarsIPM\Security\Voter\IssueAnalysisEditVoter;
use MarsIPM\Security\Voter\IssueAnalysisEditVoter;
use MarsIPM\Security\Voter\IssueAnalysisEditVoter;
use MarsIPM\Security\Voter\IssueAnalysisEditVoter;
use MarsIPM\Tests\Utils\ContainerHelperTrait;
use MarsIPM\Tests\Utils\DatabaseUtils;
use MarsIPM\Tests\Utils\ORMHelperTrait;
use MarsIPM\Tests\Utils\UserHelperTrait;
use Prophecy\PhpUnit\ProphecyTrait;
use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
use stdClass;
```
I also get asked about `TokenInterface` being ambiguous twice.

**Comment by BladeMF** [comment]
I am guessing that is has something to do with the type being both an argument and return type.

**Comment by dantleech** [comment]
Maybe as simple as deduplicating here: https://github.com/phpactor/phpactor/blob/6b0c73e9d64dd0c353d0dc3eda09159231502ee7/lib/Extension/LanguageServerCodeTransform/LspCommand/ImportAllUnresolvedNamesCommand.php#L48

**Comment by dantleech** [comment]
Fixed in #1887
